pylib/anki/stats.py

```diff
@@ -605,7 +605,7 @@ def ivlGraph(self) -> str:
             totd.append((grp, tot / float(all) * 100))
         if self.type == CARD_TYPE_NEW:
             ivlmax = 31
-        elif self.type == 1:
+        elif self.type == CARD_TYPE_LRN:
```

Comment:
this is not a card type

Reply:
Corrected all of stats to a new constant